lib/ansible/playbook/play.py

```diff
@@ -64,6 +64,9 @@ class Play(Base, Taggable, Become):
 
     # Connection
     _gather_facts        = FieldAttribute(isa='bool', default=None, always_post_validate=True)
+    _gather_subset       = FieldAttribute(isa='string', default=None, always_post_validate=True)
```

Comment:
@bcoca @jimi-c Looks like the parsing of FieldAttribute into type list (and set) is not as flexible as argument_spec parsing.  So comma separated strings are not automatically parsed into lists.  Is this something we should change at that generic level or is this something thta individual Attriubutes should handle for themselves somehow?  (Relevant code is in playbook/base.py: https://github.com/ansible/ansible/blob/devel/lib/ansible/playbook/base.py#L333  (Also note that set should probably have similar options to list.  Right now ```isa='set'``` doesn't have an equivalent to ```listof``` .)